src/controls/legend/overlayproperties.js

```diff
@@ -66,10 +118,23 @@ const OverlayProperties = function OverlayProperties(options = {}) {
           layer.setOpacity(sliderEl.valueAsNumber);
         });
       }
+      if (hasStylePicker()) {
+        styleSelection.setItems(stylePicker.map(altStyle => altStyle.title));
+        const styleSelectionEl = document.getElementById(styleSelection.getId());
+        styleSelectionEl.addEventListener('dropdown:select', (evt) => {
+          onSelectStyle(evt.target.textContent);
+        });
+        styleSelectionEl.setAttribute('aria-label', 'Välj stil');
```

Comment:
Remove these 2 rows below, since the aria-label is set when dropdown is created above.
The aria-labelledby should have been the id of  the html that is inserted by function renderStyleSelection() but since there is already a aria-label present it is enough.
```
styleSelectionEl.setAttribute('aria-label', 'Välj stil');
styleSelectionEl.setAttribute('aria-labelledby', 'Välj stil');
```